src/plugins/intel_gpu/src/graph/scatter_elements_update.cpp

```diff
@@ -52,5 +52,30 @@ std::string scatter_elements_update_inst::to_string(scatter_elements_update_node
 }
 
 scatter_elements_update_inst::typed_primitive_inst(network& network, scatter_elements_update_node const& node) : parent(network, node) {}
+void scatter_elements_update_inst::on_execute() {
+    auto input1_shape = _impl_params->input_layouts[1].get_partial_shape();
+    auto input2_shape = _impl_params->input_layouts[2].get_partial_shape();
+
+    if (!input1_shape.is_dynamic() && !input2_shape.is_dynamic()) {
```

Comment:
on execute, all inputs should not dynamic, always needs to be static. So we don't need this condition.

Reply:
Removed it. Thanks!